pkg/model/orchestrationConfig.go

```diff
@@ -60,9 +60,8 @@ type DeploymentTarget struct {
 }
 
 type AnalysisConfig struct {
-	DefaultType    string   `yaml:"defaultType,omitempty"`
-	DefaultAccount string   `yaml:"defaultAccount,omitempty"`
-	Queries        *[]Query `yaml:"queries,omitempty"`
+	DefaultMetricProvider string `yaml:"defaultMetricProvider,omitempty"`
```

Comment:
The UI does not seem to export a "default" in the YAML. Is there a configuration that would do that? Regardless, I agree with Todd's suggestion because in awesom-o it's named "defaultMetricProviderName" and having "Name" on the end will be more consistent with what's specified in the step itself.   
Example generated by the UI:
```
  analysis:
  queries:
    - metricProviderName: CDF EKS Cluster Prometheus
```